README.md

```diff
@@ -11,6 +11,15 @@ License: MIT
 
 Moved to [settings](http://cookiecutter-django.readthedocs.io/en/latest/settings.html).
 
+## Quickstart
+
+1. `docker compose -f local.yml run -d`
```

Comment:
if I'm not mistaken this will take only `local.yml` and not the  `docker-compose.yml`

Reply:
I dont understand.
In this project we have the `local.yml` file should we not use it to spin up the project locally?

Reply:
you are right, my bad